src/ui/display.c

```diff
@@ -24,9 +24,9 @@ static action_validate_cb g_validate_callback;
 static char g_amount[30];
 static char g_output_index[10];
 static char g_address[B58_ADDRESS_LEN];
-static char g_token_symbol[6];
-static char g_token_name[31];
-static char g_token_uid[65];
+static char g_token_symbol[MAX_TOKEN_SYMBOL_LEN + 1];
+static char g_token_name[MAX_TOKEN_NAME_LEN + 1];
+static char g_token_uid[1 + 2 * TOKEN_UID_LEN];
```

Comment:
These are the same values, just using the `DEFINE`d values to give context.
All of them have +1 for the null terminator (these are meant to be shown on screen) and the token is times 2 since it is shown in hex.